### PR TITLE
fixed README.md file

### PR DESCRIPTION
**Pull request by tomaszwasowicz-digica**
First attempt to fix the README.md file.

### CODE REVIEW

**Review by domenico-siciliani — APPROVED** [review]

**Comment by pawelpreczynski-digica** [comment]
could you please try to use [automatically-numbered-list](https://stackoverflow.com/questions/12145232/create-an-automatically-numbered-list)?

**Comment by tomaszwasowicz-digica** [comment]
Hey

Did you mean:

"Could WE please try to use ... " :D

Point 7.1 of:

https://docs.google.com/document/d/12r7B9mU_MEkZ0ABCrtiZZIFybSP3xFCMDliCm10LawM/edit#heading=h.8v695ab1q79a

And of course, YES WE CAN !

BR

**Comment by pawelpreczynski-digica** [comment]
YES WE CAN! ;)

File: README.md [hunk 1]
```diff
@@ -72,19 +69,19 @@
...
-  </ol>
+  </ul>
 </details>
 
 
 
 <!-- ABOUT THE PROJECT -->
-## About The Project
+
+## 1. About The Project
```
Comment (pawelpreczynski-digica) [inline README.md:83]: Please also remove numbers here on headers.

**Review by pawelpreczynski-digica — CHANGES_REQUESTED** [review]

**Review by pawelpreczynski-digica — APPROVED** [review]

**Pull request merged**